External Dependencies/AutoHotkey-JSON-master/JSON.ahk

```diff
@@ -83,7 +83,7 @@ class JSON
 					; the 'IsArray' property. If so, Array() will be called normally,
 					; otherwise, use a custom base object for arrays
 						static json_array := Func("Array").IsBuiltIn || ![].IsArray ? {IsArray: true} : 0
-					
```

Comment:
All changes in this file are whitespace only. Probably something with the IDE used to open it. Revert this since it's not relevant.

Reply:
Welp I've tried like 4 different things and none have worked. If this wasn't a pet project I'd try more but it's not worth the time.